rs/consensus/src/idkg.rs

```diff
@@ -488,111 +479,91 @@ impl IDkgPriorityFnArgs {
             active_transcripts.insert(transcript_ref.transcript_id);
         }
 
-        let (certified_height, requested_signatures) = state_reader
-            .get_certified_state_snapshot()
-            .map_or(Default::default(), |snapshot| {
-                let request_contexts = snapshot
-                    .get_state()
-                    .signature_request_contexts()
-                    .values()
-                    .flat_map(get_context_request_id)
-                    .collect::<BTreeSet<_>>();
+        let (certified_height, _) =
+            state_reader
+                .get_certified_state_snapshot()
+                .map_or(Default::default(), |snapshot| {
+                    let request_contexts = snapshot
+                        .get_state()
+                        .signature_request_contexts()
+                        .values()
+                        .flat_map(get_context_request_id)
+                        .collect::<BTreeSet<_>>();
 
-                (snapshot.get_height(), request_contexts)
-            });
+                    (snapshot.get_height(), request_contexts)
+                });
 
         Self {
             finalized_height: block_reader.tip_height(),
             certified_height,
```

Comment:
```suggestion
            certified_height: state_reader.latest_certified_height(),
```
Then we can remove the assignment above